packages/react-vapor/src/components/menu/MenuConnected.tsx

```diff
@@ -49,12 +52,13 @@ const mapDispatchToProps = (dispatch: IDispatch, ownProps: IMenuOwnProps): IMenu
 @ReduxConnect(mapStateToProps, mapDispatchToProps)
 export class MenuConnected extends React.Component<IMenuProps, {}> {
     private list: HTMLDivElement;
-    private button: HTMLButtonElement;
+    private button: HTMLDivElement;

```

Comment:
It's not a button anymore tho 🤔

Reply:
Nope, but I target the button with a div element so I keep the name button to explain what I want to calculate